src/Updown.ts

```diff
@@ -6,12 +6,12 @@ import * as qs from 'querystring';
  * Options object used when creating a new Updown API client
  */
 export interface Options {
-	readOnly? : boolean;
+  readOnly?: boolean;
 }
 
 /** An object with primitive values (i.e. no arrays/objects) */
 interface PrimitiveObject {
-	[key : string] : string | number;
+  [key: string]: string | number;
```

Comment:
There should always be a space before *and* after the `:` for type notations, as in the original code. This is to more easily see the difference between `{ prop : type }` and `{ prop: value }` even out of context.